prometheus/metric.go

```diff
@@ -24,6 +24,8 @@ import (
 
 const separatorByte byte = 255
 
+var separatorByteSlice = []byte{255} // For convenient use with xxhash.
```

Comment:
not that it's likely to change, but should this not be `[]byte{separatorByte}`

Reply:
Yeah, that's better. (Originally, I wanted to remove separatorByte altogether, but then realized I want to keep the old hashing for vectors.) Will sent you a PR.